src/api/entities/Proposal/types.ts

```diff
@@ -51,10 +20,25 @@ export enum ProposalStage {
   Ended = 'Ended',
 }
 
-export interface ProposalDetails {
-  state: ProposalState;
-  module: string;
+export interface Call {
   method: string;
+  module: string;
+}
+
+export interface ProposalDetails {
+  pipId: number;
+  proposer: string;
+  createdAt: number;
+  url: string;
+  description: string;
+  coolOffEndBlock: number;
+  endBlock: number;
+  call?: Call;
```

Comment:
This should be `transaction: TxTag`

Reply:
Make sure when you're converting that the method and module strings follow the same format we use in TxTags